cmd_init_wallet.go

```diff
@@ -45,7 +45,7 @@ type secretSourceFile struct {
 
 type secretSourceK8s struct {
 	Namespace             string `long:"namespace" description:"The Kubernetes namespace the secret is located in"`
-	SecretName            string `long:"secret-name" description:"The name of the Kubernetes secret"`
+	Name                  string `long:"name" description:"The name of the Kubernetes secret"`
```

Comment:
Hmm, while I see why we'd want to clean this up, I don't think it's worth breaking anyone already using the previous flag name. Perhaps a little bit of copy/paste here (meaning: don't rename the `k8sSecretOptions` but instead create a new one for the config map) would go a long way. Would also reduce the diff quite a bit.

Reply:
Okay, I updated things so that we can avoid renaming any CLI flags ✅ Still did some translating from the secret/configmap specific structs to this common "object" struct prior to passing to (save/read)k8s. Let me know if this is any better!